deployment/articles-dxp/01-saml/06-configuring-saml.markdown

```diff
@@ -56,3 +62,70 @@ Note also the introduction of the *SAML KeyStoreManager Implementation
 Configuration* in *Control Panel* &rarr; *System Settings* &rarr; Security
 &rarr; SSO. The options for this configuration are explained above in the
 Setting up @product@ as a SAML Identity Provider section.
+
+In the latest version of the plugin, the SHA256 algorithm is the default
+encryption algorithm used in the configuration and to generate keys. The default
+configuration tries SHA256, then SHA384, then SHA512 before falling back to
+SHA1. Because SHA1 is potentially vulnerable, you can blacklist it using this
+property: 
+
+```properties
+blacklisted.algorithms=["blacklisted_algorithm_url", "another_blacklisted_algorithm_url"]
+```
```

Comment:
Add: name of the property config file: `com.liferay.saml.opensaml.integration.internal.bootstrap.SecurityConfigurationBootstrap.config`

Reply:
Done.